api/v1/tetragon/events.proto

```diff
@@ -80,6 +80,9 @@ message Filter {
   google.protobuf.BoolValue in_init_tree = 16;
   // Filter ancestor processes' binaries using RE2 regular expression syntax.
   repeated string ancestor_binary_regex = 17;
+  // Filter by the container name in the process.pod.container field using RE2 regular expression syntax:
+  // https://github.com/google/re2/wiki/Syntax
+  repeated string container_name = 18;
```

Comment:
nit: maybe add the `_regex` suffix like others (I see that container_id doesn't but maybe that's a better idea to be explicit)
```suggestion
  repeated string container_name_regex = 18;
```

Reply:
Pushed :)